src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/MvcExtensibilityConfiguration.cs

```diff
@@ -51,10 +58,12 @@ public override bool Equals(ProjectExtensibilityConfiguration other)
             }
 
             // We're intentionally ignoring the 'Kind' here. That's mostly for diagnostics and doesn't influence any behavior.
-            return Enumerable.SequenceEqual(
-                Assemblies.OrderBy(a => a.Identity.Name).Select(a => a.Identity),
-                other.Assemblies.OrderBy(a => a.Identity.Name).Select(a => a.Identity),
-                AssemblyIdentityEqualityComparer.NameAndVersion);
+            return 
+                Enumerable.SequenceEqual(
+                    Assemblies.OrderBy(a => a.Identity.Name).Select(a => a.Identity),
+                    other.Assemblies.OrderBy(a => a.Identity.Name).Select(a => a.Identity),
+                    AssemblyIdentityEqualityComparer.NameAndVersion) &&
+                RazorLanguageVersion == other.RazorLanguageVersion;
```

Comment:
Was this just a bug?

Reply:
No, just formatted and added the `RazorLanguageVersion` check to the equality.

Reply:
Ok gotcha

Reply:
nit: can you put the language version first? That should be very cheap to compare.

Reply:
another non-nit. Either make `RazorLanguageVersion` implement `IEquatable<>` or use object.ReferenceEquals(...). Using `==` with non-simple types is non-obvious what it does.